PS4k/src/com/vitco/layout/content/menu/MainMenuLogic.java

```diff
@@ -532,6 +534,59 @@ protected Object doInBackground() throws Exception {
 
         // ---------------
 
+        // add "turntable image sequence" export
+        FieldSet turnTableImageSeq = new FieldSet("turntable_image_sequence", "Turntable Render (*.png)");
+        turnTableImageSeq.addComponent(new LabelModule("Render as multiple image sequence"));
+        final TextInputModule frameCount = new TextInputModule("frames_count", "Number of frames", "60", true);
+        frameCount.setVisibleLookup("export_type=turntable_image_sequence");
+
+        turnTableImageSeq.addComponent(frameCount);
+        turnTableImageSeq.addComponent(new CheckBoxModule("watermark", "Add watermark: Show VoxelShop some love", true));
+        turnTableImageSeq.addComponent(new SeparatorModule("Animation orbit:"));
+        turnTableImageSeq.addComponent(new ComboBoxModule("animation_orbit", new String[][]{
+                new String[]{"clockwise", "ClockWise"},
+                new String[]{"anti-clockwise", "Anti-Colockwise"},
+        }, 0));
+        turnTableImageSeq.addComponent(new SeparatorModule(""));
+        turnTableImageSeq.addComponent(new CheckBoxModule("add_background_color", "Add Image background color", false));
+        TextInputModule TISBackgroundColor = new TextInputModule("background_color", "Image Background Color: e.g #ffffff", "#000000", true);
+        TISBackgroundColor.setVisibleLookup("export_type=turntable_image_sequence&turntable_image_sequence.add_background_color=true");
```

Comment:
I didn't hide it at first but later i thought it will be easier to understand cause if there is no background color the image background  will be transparent. And also other applications does that,but if you don't like it, i can change it.